ml-algorithms/src/main/java/org/opensearch/ml/engine/memory/MLMemoryManager.java

```diff
@@ -176,7 +176,7 @@ void innerGetFinalInteractions(String conversationId, int lastNInteraction, Acti
         searchRequest.source(searchSourceBuilder);
 
         searchRequest.source().size(lastNInteraction);
-        searchRequest.source().sort(ConversationalIndexConstants.INTERACTIONS_CREATE_TIME_FIELD, SortOrder.DESC);
+        searchRequest.source().sort(ConversationalIndexConstants.INTERACTIONS_CREATE_TIME_FIELD, SortOrder.ASC);
 
```

Comment:
@Zhangxunmt this change may break `getFinalInteractions`, based on the parameter of this method, it fetch latest N interactions. After this change, it will fetch first N interactions.

Reply:
I would like to keep using DESC and make it sort as ASC in line 186 with `result.add(0, Interaction.fromSearchHit(hit));`